docs/core/diagnostics/trace-perfcollect-lttng.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Tracing .NET applications with LTTng.
+description: A tutorial that walks you through collecting a trace with LTTng and perfcollect in .NET.
+ms.topic: tutorial
+ms.date: 10/23/2020
+---
+
+# Tracing .NET applications with LTTng and PerfCollect
+
+**This article applies to: ✔️** .NET Core 2.1 SDK and later versions
+
+When a performance problem is encountered on Linux, collecting a trace with `perfcollect` can be used to gather detailed information about what was happening on the machine at the time of the performance problem.
+
+`perfcollect` is a bash script that leverages [Linux Tracing Tookit-Next Generation (LTTng)](https://lttng.org) to collect events written from the runtime or any [EventSource](xref:System.Diagnostics.Tracing.EventListener), as well as [perf](https://perf.wiki.kernel.org/) to collect CPU samples of the target process.
+
+## Preparing Your Machine
+
+Follow these steps to prepare your machine to collect a performance trace with `perfcollect`.
```

Comment:
I would add a warning here that if you are using a container, it will need to have the SYS_ADMIN capability. There is a more a generic callout about containers far below, but for anyone following the steps in order this is a requirement they need to act on now.